VISION.md

```diff
@@ -0,0 +1,86 @@
+# API Improvement Proposals: Mission and Vision
+
+## Mission
+
+Help organizations build consistent, uniform remote application programming
+interfaces and clients.
+
+## Vision
+
+API Improvement Proposals help producers of APIs delight their users with API
+styleguide tooling, API best practices and validation tooling, and client
+generators.
+
+AIPs provide an ecosystem of extensible tooling and guidance that can bring
+maturity to an existing API governance program or bootstrap a new one. In order
+to facilitate this tooling ecosystem, AIPs provide guidance about
+resource-oriented API design.
+
+The AIPs can be categorized around layers of projects, with each higher layer
+building on top of the lower layers.
+
+### Layer 1: API style guide tooling
+
+As an organization exposes multiple APIs to customers, the value of consistency
```

Comment:
```suggestion
As an organization exposes more APIs to customers, the value of consistency
```

The value of consistency increasing when going from single to multiple (i.e. from 1 to 2) isn't a bit binary - which I don't think is the intention.